src/components/Body/MovieCrads/MovieCard/MovieCard.tsx

```diff
@@ -8,10 +10,15 @@ type MoviePropsType = {
 };
 
 function MovieCard({ movie }: MoviePropsType) {
```

Comment:
![image](https://user-images.githubusercontent.com/106384184/208587636-bceaa494-7ef2-46d0-a0b4-614cff90da7b.png)
Icon should appear on hover